src/surfing_penguin/routes.py

```diff
@@ -1,19 +1,26 @@
 """routes.py: Each function in this file indicates a web page (HTML page)."""
 
-import datetime
-from surfing_penguin import surfing_penguin, api, session
+from surfing_penguin import surfing_penguin, api
 from flask import render_template, flash, redirect, url_for, request
 from flask_restplus import Resource, fields
 from surfing_penguin.forms import LoginForm
-from surfing_penguin.db_interface import Qstnr
-from surfing_penguin.models import User
+from surfing_penguin.db_interface import Qstnr, User_func
 
 question = api.model("question_model", {
         'content': fields.String,
         'id': fields.Integer
     })
 
 
```

Comment:
Descriptions for API models should be added. Also, API models for POST functions are missing.